app/views/user_mailer/more_than_10_errors.text.haml

```diff
@@ -0,0 +1 @@
+This is a email notifying you that 10 errors occurred in last hour.
```

Comment:
This email is not helpful at all. You need to show some summary of the errors occurred. It can be just the message from each error. No need to show stacktrace of other details. Also, add links to those errors.